ote_sdk/ote_sdk/usecases/exportable_code/demo/demo_package/model_container.py

```diff
@@ -110,7 +109,10 @@ def infer(self, frame):
         """
         # getting result include preprocessing, infer, postprocessing for sync infer
         predictions, frame_meta = self.core_model(frame)
-        predictions = detection2array(predictions)
+
+        # MaskRCNN returns tuple so no need to process
+        if not self.segm:
```

Comment:
```suggestion
        if self._task_type == TaskType.DETECTION:
```

If I'm correctly understanding, detection2array() is only for detection task.